frontend/src/layouts/Layout.jsx

```diff
@@ -42,24 +45,30 @@ const Layout = () => {
         </Grid>
         <Grid item my={12} lg={12}>
           <BCBox>
-            <Paper p={2} elevation={5} sx={{ padding: "1rem", minHeight: '5vh' }}>
-              <Breadcrumbs separator={<NavigateNextIcon fontSize="small" />}>
-                {crumbs.map((crumb, index) =>
-                  index + 1 !== crumbs.length ? (
-                    <Link
-                      key={crumb.path}
-                      component={RouterLink}
-                      to={crumb.path}
-                      disabled={true}
-                    >
-                      {crumb.label}
-                    </Link>
-                  ) : (
-                    <span key={crumb.path}>{crumb.label}</span>
-                  ),
-                )}
-              </Breadcrumbs>
-            </Paper>
+            {crumbs && (
+              <Paper
+                p={2}
+                elevation={5}
+                sx={{ padding: '1rem', minHeight: '5vh' }}
+              >
+                <Breadcrumbs separator={<NavigateNextIcon fontSize="small" />}>
+                  {breadcrumbs.map((crumb, index) =>
```

Comment:
As we set up and add the Organizations and [Organization] pages to the routes, the breadcrumbs should be updated accordingly.

Since we dont have those pages yet, and that these crumbs will be added as those pages come, I think we can leave this for now and adjust if needed when those pages are made.

thoughts?